.vscode/launch.json

```diff
@@ -1,123 +1,24 @@
 {
```

Comment:
Why did you remove most of the configurations?

Reply:
Didn't know you used it ;). It gave deprecation warnings, so i decided to regenerate it. I don't mind either way, shall i undo this change?

Reply:
Please do. I'll see if I get the deprecation warnings, too, and then update it. My idea would be to have some ready-to-go launch configs, so anyone who would be working on this project could immediately get started.

Reply:
Done